packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -0,0 +1,62 @@
+Dropdown containers display a menu with additional items as soon as a user clicks on the trigger element, e.g. a dropdown button. Clicking on one of the items in the dropdown performs the action. We typically use dropdowns to allow users to select one option from a list of available choices.
+
+For information about the trigger element that opens the dropdown, please refer to our [dropdown button](buttons/dropdown-button.md) and [split button](buttons/split-button.md) components.
```

Comment:
Any element can be set as trigger for a dropdown. Dropdown button and split button are just two special cases that have the dropdown functionality baked into them.

Reply:
```suggestion
```